instrumented/nightly-tests/src/main/kotlin/com/datadog/android/nightly/server/LocalServer.kt

```diff
@@ -0,0 +1,57 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.nightly.server
+
+import io.ktor.application.call
+import io.ktor.application.install
+import io.ktor.features.ContentNegotiation
+import io.ktor.gson.gson
+import io.ktor.response.respondRedirect
+import io.ktor.routing.get
+import io.ktor.routing.routing
+import io.ktor.server.engine.ApplicationEngine
+import io.ktor.server.engine.embeddedServer
+import io.ktor.server.netty.Netty
+
+internal class LocalServer {
+
+    var engine: ApplicationEngine? = null
+
+    fun start(redirection: String) {
+        engine = embeddedServer(Netty, PORT) {
+            install(ContentNegotiation) {
+                gson {}
+            }
+            routing {
+                get(PATH) {
+                    call.respondRedirect(redirection, true)
+                }
+            }
+        }
+        engine?.start(wait = false)
+    }
+
+    fun stop() {
+        Thread {
+            engine?.stop(SHUTDOWN_MS, STOP_TIMEOUT_MS)
+        }.start()
```

Comment:
I'm just curious why it is done on another thread - to avoid blocking main thread during `stop` request?

Reply:
Not really sure, I just copy pasted that class from the sample app :)